packages/material-ui/src/ButtonBase/ButtonBase.js

```diff
@@ -89,20 +77,19 @@ class ButtonBase extends React.Component {
 
   handleContextMenu = createRippleHandler(this, 'ContextMenu', 'stop');
 
-  handleBlur = createRippleHandler(this, 'Blur', 'stop', () => {
-    clearTimeout(this.focusVisibleTimeout);
+  handleBlur = createRippleHandler(this, 'Blur', 'stop', event => {
```

Comment:
Doe createRippleHandler forward the event?

Reply:
Yes, it should.